css/properties/azimuth.json

```diff
@@ -0,0 +1,56 @@
+{
+  "css": {
+    "properties": {
+      "azimuth": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/azimuth",
+          "support": {
+            "chrome": {
+              "version_added": null
```

Comment:
This should be `false`. So should Chrome Android, Android webview, and both flavors of Opera.

Reply:
Thanks, fixed in 3280712.